website/index.html

```diff
@@ -5,6 +5,26 @@
 </head>
 <body>
     <h1>Xbox controller panel</h1>
+   <ul>
+    <li><b>Left Trigger</b>:Move Backward</li>
+    <li><b>Left Bumper</b>:Belt</li>
+    <li><b>L3</b>:Preset 1</li>
+    <li><b>Right Trigger</b>:Move Forward</li>
+    <li><b>Right Bumper</b>:Move Forward Slowly</li>
+    <li><b>R3</b>:Preset 2</li>
+    <li><b>Select</b>:Start Dig Autonomy</li>
+    <li><b>Start</b>:Preset 3li>
```

Comment:
"Camera Preset 3" 
Also there looks to be a typo: <li><b>Start</b>:Preset 3</li>